packages/components/src/components/gradient-background/gradient-background.tsx

```diff
@@ -16,10 +16,12 @@ const GradientBackground: React.FC<React.PropsWithChildren<TProps>> = ({
     tertiary,
 }) => (
     <div className='gradient-background' style={{ backgroundColor: color }}>
-        <div className='gradient-background__container'>
-            <div className='gradient-background__primary' style={{ background: primary }} />
-            <div className='gradient-background__secondary' style={{ background: secondary }} />
-            <div className='gradient-background__tertiary' style={{ background: tertiary }} />
+        <div className='gradient-background__filter'>
+            <div className='gradient-background__container'>
```

Comment:
@heorhi-deriv Couldn't give the filter to the container class to avoid adding another div element? 🤔

Reply:
@farzin-deriv i thought about that, but container has max width and the blur effect will apply only for this area.